mllib/src/main/scala/org/apache/spark/ml/classification/FMClassifier.scala

```diff
@@ -186,7 +186,7 @@ class FMClassifier @Since("3.0.0") (
   @Since("3.0.0")
   def setSeed(value: Long): this.type = set(seed, value)
 
-  override protected[spark] def train(
+  override protected def train(
```

Comment:
Oh why was this an issue? out of curiosity.

Reply:
The super class has ```protected def train(dataset: Dataset[_])```. When overriding this, I thought it's not supposed to widen the scope? The other classifiers have 
```override protected def train```

Reply:
OK that's fine; this hasn't even been released anyway. Consistency is good.
(There are use cases for widening access but nothing applicable here I think)